.github/workflows/ci-cd.yaml

```diff
@@ -324,7 +295,7 @@ jobs:
                   build/trento-s390x.tgz
 
   release-rolling:
-    needs: build-static-binary
+    needs: [build-static-binary, test-binary, test-checks]
```

Comment:
Yes, that's a discussion I think we need to have.
I'd be ok with what you suggest, would like to have an agreement on this.

The reasoning was this: we release rolling independently from tests e2e so even if they fail, we can have an environment where to test it and debug possibly.

But again, let's discuss :wink:

Reply:
While I think we should block the rolling-release (or any other release) when the e2e tests are failing, I'd would still like to see the demo environment getting an update with the latest release: If we are at the point where we missed to see failed test before accepting a PR, at least having an environment where we can look at the issue and debug it could be useful. 

Right now that's tricky since `deploy-server` depends on `release-rolling` so if we change this, I'd readjust the `deploy-server` dependencies too.

Reply:
I think the staging deployment AND the rolling release should be blocked if any of the tests including e2e tests fail. To replicate the failing scenario we can replay the scenario locally with photofinish, we don;t need to break demo for this.

Reply:
So, this is how it looks now
https://github.com/trento-project/trento/actions/runs/1722381117

Basically `test-e2e` becomes required by all of the following steps.

Should be enough for now. Is this ok with you guys?